isthmus/src/test/java/io/substrait/isthmus/SubqueryPlanTest.java

```diff
@@ -0,0 +1,117 @@
+package io.substrait.isthmus;
+
+import static io.substrait.isthmus.PlanTestBase.asString;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import com.google.protobuf.util.JsonFormat;
+import io.substrait.proto.Plan;
+import java.io.IOException;
+import java.util.Arrays;
+import org.apache.calcite.sql.parser.SqlParseException;
+import org.junit.jupiter.api.Test;
+
+public class SubqueryPlanTest {
+  // TODO: Add a roundtrip test once the ProtoRelConverter is committed and updated to support
+  // subqueries
+  @Test
+  public void existsCorrelatedSubquery() throws IOException, SqlParseException {
+    SqlToSubstrait s = new SqlToSubstrait();
+    String[] values = asString("tpch/schema.sql").split(";");
+    var creates = Arrays.stream(values).filter(t -> !t.trim().isBlank()).toList();
+    Plan plan =
+        s.execute(
+            "select l_partkey from lineitem where exists (select o_orderdate from orders where o_orderkey = l_orderkey)",
+            creates);
+    String planText = JsonFormat.printer().includingDefaultValueFields().print(plan);
+
+    assertTrue(planText.contains("setPredicate"));
```

Comment:
I'd prefer to see structured verification as opposed to text verification here. Any reason not to?

Reply:
I changed to structured verification.  The text verification is easier to write, but fail to check the plan structure.